plugins/device/init.go

```diff
@@ -32,12 +32,22 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Version:      1,
 			RequestType:  utils.GetTypeString((*deviceapi.AddDeviceReq)(nil)),
 			ResponseType: utils.GetTypeString((*deviceapi.AddDeviceResp)(nil)),
-			HandlerFunc:  deviceAddHandler},
+			HandlerFunc:  deviceAddHandler,
+		},
+		route.Route{
+			Name:         "EditGroup",
+			Method:       "POST",
+			Pattern:      "/peers/{peerid}/group/{group_id}",
+			Version:      1,
+			ResponseType: utils.GetTypeString((*deviceapi.EditGroupResp)(nil)),
+			HandlerFunc:  groupEditHandler,
```

Comment:
`peerGroupEditHandler`

Same comment elsewhere. Prefix peer wherever applicable. Without that, someone new might mistake the group to be related to device and not peer.